src/main/scala/uk/gov/hmrc/playcrosscompilation/AbstractPlayCrossCompilation.scala

```diff
@@ -50,6 +50,11 @@ abstract class AbstractPlayCrossCompilation(
       }
     )
 
+  def playCrossScalaBuilds(scalaVersions: Seq[String]) = playVersion match {
+    case Play25 => scalaVersions.filter(version => version.startsWith("2.11") || version.startsWith("2.10"))
```

Comment:
Is play 2.5 available for scala 2.10?

Reply:
I don't think so 
> Play 2.5 has dropped support for Scala 2.10 
[Migration Guide](https://www.playframework.com/documentation/2.6.x/Migration25#Scala-2.10-support-discontinued)

Good catch - this was just i was uncertainty - would clean up in minute.